File: CruiseControl_NET/rev6234-6870/left-trunk-6870/project/core/util/ILogger.cs
using System;
namespace ThoughtWorks.CruiseControl.Core.Util
{
    public interface ILogger
    {
        void Debug(string message, params object[] values);
        void Info(string message, params object[] values);
        void Warning(string message, params object[] values);
        void Warning(Exception error);
        void Error(string message, params object[] values);
        void Error(Exception error);
    }
}

File: SQuirrel_SQL/rev5082-5155/right-trunk-5155/fw/src/net/sourceforge/squirrel_sql/fw/util/log/ILogger.java
package net.sourceforge.squirrel_sql.fw.util.log;

import org.apache.log4j.Level;



public interface ILogger
{
	public void info(Object message);
	public void info(Object message, Throwable th);
	public void warn(Object message);
	public void warn(Object message, Throwable th);
	public void error(Object message);
	public void error(Object message, Throwable th);
	public void debug(Object message);
	public void debug(Object message, Throwable th);

	boolean isDebugEnabled();
	boolean isInfoEnabled();
    
    
    void setLevel(Level l);
}

